modules/installation-osp-kuryr-settings.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-kuryr-settings_{context}"]
+= Adjusting Kuryr settings in {rh-openstack} deployments
+
+If you are deploying on {rh-openstack} with `Kuryr` there are several options
+available for tuning of how Kuryr will behave in the cluster. This will allow
+you to find balance between OpenStack Neutron ports usage and how quickly Kuryr
+is able to handle new pods being created.
+
+You can adjust Kuryr settings before the installation as well as modify them
+later.
+
+.Port pools
+
+The Kuryr settings available for adjusting are related to *port pools* feature.
+The goal of that function is to minimize interactions with OpenStack Neutron
+in order to speed up creation of the new pods. Without port pools Kuryr would
+need to request a Neutron port creation each time a new Pod is spawned and
+port deletion when Pod gets removed. Those are relatively expensive operations,
+and the idea of port pools is to do them asynchronously whenever possible.
+
+The feature makes Kuryr keep a _pool_ of Neutron ports ready to be connected to
+Pod on creation. Neutron ports used by Kuryr are created in a subnet and
+attached as subports to a trunk port - the OpenShift node port. As Kuryr
+keeps each namespace in a separate subnet, it effectively means that a
+separate port pool is kept for each namespace-worker pair.
```

Comment:
```suggestion
The Neutron ports that Kuryr users are created in [a common || their own] subnet and are attached as subports to a trunk port: the {product-title} cluster node port. 

Because Kuryr keeps each namespace in a separate subnet, a separate ports pool is maintained for each namespace-worker pair. 
```

Fill in the brackets, there. :)